src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -31,14 +32,16 @@
     private final String address;
     private final List<JsonAdaptedTag> tagged = new ArrayList<>();
     private final String birthday;
+    private final String pin;
 
     /**
      * Constructs a {@code JsonAdaptedPerson} with the given person details.
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged, @JsonProperty("birthday") String birthday) {
+            @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
+                             @JsonProperty("birthday") String birthday, @JsonProperty("pin") String pin) {
```

Comment:
should the parameters be aligned? eg. line 44 reduce indent to align with line 43?